index.html

```diff
@@ -679,7 +288,45 @@ <h1 class="foot-title">
         </ul>
       </div>
     </footer>
+
+     <!-- <div class="popup" id="popup">
+      <div class="popup__bg">
+        <div class="popup__header">
+          <button class="popup__close" id="btnClosePopup">
+            <span class="material-symbols-outlined">close</span>
+          </button>
+        </div>
+        <div class="popup__main">
+          <div>
+            <h3 class="popup__title clr-blue"></h3>
+            <ul class="popup__technologies technologies technologies--left"></ul>
+          </div>
+          <div class="popup__external popup__external--desktop">
+            <a href="" target="_blank" class="popup__link popup__link--deploy">
+              <span>See Live</span>
+              <object data="./img/icons/external.svg"></object>
+            </a>
+            <a href="" target="_blank" class="popup__link popup__link--code">
+              <span>See Source</span>
+              <object data="./img/icons/github-white.svg"></object>
+            </a>
+          </div>
+        </div>
+        <p class="popup__description description"></p>
+        <div class="popup__external popup__external--mobile">
+          <a href="" class="popup__link popup__link--deploy">
+            <span>See Live</span>
+            <object data="./img/icons/external.svg"></object>
+          </a>
+          <a href="" class="popup__link popup__link--code">
+            <span>See Source</span>
+            <object data="./img/icons/github-white.svg"></object>
+          </a>
+        </div>
+      </div>
+    </div> -->
```

Comment:
- [ ] Well done using JavaScript to populate this section as requested by the previous CR. Please 🙏🏽 remove the unused code for clean code and good practices of HTML & CSS, we use the comments to describe what the code does to help other programmers to read our code.